### PR TITLE
Standalone Planner

### PR DESCRIPTION
**Pull request by michaelkleinhenz**
This adds the new standalone header to the Planner runtime, making it a full standalone Planner.

### CODE REVIEW

File: runtime/src/assets/stylesheets/shared/_variables.less [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import (reference) '../../../../node_modules/patternfly/src/less/variables';
+@import (reference) '../../../../node_modules/patternfly/dist/less/variables';
```
Comment (SMahil) [inline runtime/src/assets/stylesheets/shared/_variables.less]: Is there any specific reason to change this path?

Comment (debloper) [reply runtime/src/assets/stylesheets/shared/_variables.less]: Distributables should be distributed from `dist`
It's crazy that it was using `src` in the first place, and somehow slipped through the code-reviews.

Comment (michaelkleinhenz) [reply runtime/src/assets/stylesheets/shared/_variables.less]: Yes, and it was not working anymore with my setup. I think some step in the build stripped the src from the dependency.

**Comment by fabric8cd** [comment]
@michaelkleinhenz fabric8/fabric8-ui:SNAPSHOT-PR-2378-2 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2378-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@michaelkleinhenz fabric8/fabric8-ui:SNAPSHOT-PR-2378-4 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2378-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@michaelkleinhenz fabric8/fabric8-ui:SNAPSHOT-PR-2378-5 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2378-fabric8-planner.badger.fabric8.io

**Comment by maxandersen** [comment]
> Planner runtime, making it a full standalone Planner.

lets please not call this full standalone planner as that implies the backend is fully standalone too which this PR don't change. 

Also, maybe just me but I and i've seen others too get utterly confused  when we use the term "planner runtime" in the frontend code. 

could we at least call it "planner UI runtime" or "planner UI wrapper" or something to disambiguate these terms/concepts ? Thanks.

**Comment by maxandersen** [comment]
i haven't found traces of it elsewhere but with this change - how are updates to components supposed to happen/be aligned ?

**Comment by maxandersen** [comment]
and to be clear - i'm not saying what is in the PR is wrong - just looking for the implications and process changes needed to do this change as it probably have bigger impact there than the code (at least initially)

**Comment by michaelkleinhenz** [comment]
@maxandersen the change here is transparent for now. When we have the Planner UI running and deploying standalone, then we can start applying the changes to Platform. We can (and will need to) support both ways of deploying Planner UI for a while until Platform has integrated it.

For the CI/CD integration, Pranav is working on this already.

**Comment by debloper** [comment]
@maxandersen I am with you in calling it `wrapper` instead of `runtime`. Apart from that, making it as standalone as possible is probably the way to go, even if not in this patch/PR.

If the user chooses to deploy it on their own server, opting to use custom auth/no-auth backend, or even, no backend but a static webapp with the structured data stored in IndexedDB - that'll be something which separates us from most other planning tool and actually puts us a few miles ahead.

I'd look forward to see/make that happen.

**Comment by fabric8cd** [comment]
@michaelkleinhenz fabric8/fabric8-ui:SNAPSHOT-PR-2378-10 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2378-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@michaelkleinhenz fabric8/fabric8-ui:SNAPSHOT-PR-2378-12 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2378-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@michaelkleinhenz fabric8/fabric8-ui:SNAPSHOT-PR-2378-13 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2378-fabric8-planner.badger.fabric8.io

**Pull request merged**